package.json

```diff
@@ -1201,6 +1218,11 @@
         "view": "confluent-flink-database",
         "contents": "No UDFs found.\n[Generate a new Flink UDF from template](command:confluent.artifacts.scaffold)",
         "when": "config.confluent.flink.artifacts && confluent.ccloudConnectionAvailable && confluent.flinkDatabaseSelected && confluent.flinkDatabaseViewMode == 'UDFs'"
```

Comment:
This one needs to include the search context value too since we'll see both empty states for an unmatched search string:

<img width="553" height="195" alt="Image" src="https://github.com/user-attachments/assets/c32e4c56-c337-42d2-a92c-9e2def2207fa" />


```suggestion
        "contents": "No UDFs found.\n[Generate a new Flink UDF from template](command:confluent.artifacts.scaffold)",
        "when": "config.confluent.flink.artifacts && confluent.ccloudConnectionAvailable && confluent.flinkDatabaseSelected && confluent.flinkDatabaseViewMode == 'UDFs' && !confluent.flinkDatabaseSearchApplied"
```